app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -37,8 +37,8 @@ const ExperimentDetails: React.FC = () => {
         </div>
       </LabelledRow>
       <LabelledRow label="Hypothesis" value={data.objectives} />
-      <LabelledRow label="Feature" />
-      <LabelledRow label="Audience" />
+      <LabelledRow label="Feature" value={data.features.toString()} />
+      <LabelledRow label="Audience" value={data.audience} />
```

Comment:
Are these the labels or the values? We should be showing the labels, not the values.

Reply:
oooh you're right you're right